docs/csharp/programming-guide/unsafe-code-pointers/how-to-obtain-the-address-of-a-variable.md

```diff
@@ -29,11 +29,12 @@ int* p = &number; //address-of operator &
  You cannot get the address of a constant or a value.  
```

Comment:
not related to your change, but can you add a csharp identifier on line 20?

Reply:
@mairaw I've also made minor style fixes. Hope all of them are correct.